rerun_py/rerun_sdk/rerun/__init__.py

```diff
@@ -216,6 +226,32 @@ def set_enabled(enabled: bool) -> None:
     bindings.set_enabled(enabled)
 
 
+def strict_mode() -> bool:
+    """
+    Strict mode enabled.
+
+    In strict mode, misuses of the Rerun API will result in exception being raised.
```

Comment:
not happy with calling it `misuses`, leaves too many open questions. More like above `incorrect use of the Rerun API (wrong parameter types etc.)`